Plugin.pm

```diff
@@ -1208,8 +1245,25 @@ sub _trackItem {
 		$item->{line1} .= ' (' . cstring($client, 'PLUGIN_QOBUZ_HIRES') . ')';
 	}
 
-	if ( $track->{work} ) {
-		$item->{work} = $track->{work};
+	# Enhancements to work/composer display for classical music (tags returned from Qobuz are all over the place)
+	if ( $track->{album}->{isClassique} ) {
```

Comment:
As outlined in a previous comment I believe you should evaluate whether an album is classical or not only here. Otherwise you might be reading a stale `isClassique` flag in case the user defined list of classical genes has changed.

Reply:
See above.